src/main/java/com/openshift/internal/restclient/model/BuildConfig.java

```diff
@@ -142,6 +200,14 @@ public void setOutput(DockerImageURI imageUri){
 		return null;
 	}
 
+	private void storeEnvironmentVars(final String [] key, ModelNode root, Map<String, String> envs){
```

Comment:
@aslakknutsen consider making this variable accept a key that will be used to look up a path similiar to the existing convenience "get" methods